test/helpers.js

```diff
@@ -86,6 +106,18 @@ const nonceAndId = (nonce, id) => {
   return Ethers.utils.hexZeroPad(Ethers.utils.hexlify(nonce), 8) + Ethers.utils.hexZeroPad(Ethers.utils.hexlify(id), 1).substr(2)
 }
 
+function generateDeposit(targetChainID = 0, secret = 31) {
```

Comment:
Do we expect deposits to use a variable amount of randomness for the secret?

Reply:
Yes. We could do this differently, but for now this works and makes sure there are not repeat commitments

Reply:
what do you mean @nepoche ?